screens/popups/qr_read/qr_read.gui_script

```diff
@@ -0,0 +1,101 @@
+local monarch = require "monarch.monarch"
+local share = require "pokedex.share"
+local url = require "utils.url"
+local _file = require "utils.file"
+local messages = require "utils.messages"
+local screens = require "utils.screens"
+
+local function stop_capture(self)
+	if self.cameraframe == nil then
+		return
+	end
+
+	self.cameraframe = nil
+	camera.stop_capture()
+end
+
+
+local function start_capture(self)
+	if not camera then
+		gui.set_text(self.label, "Capture Status: UNAVAILABLE")
```

Comment:
I get this when running on my PC. Intentional? Should I even be able to launch this screen on my PC?

Reply:
Oh, I figured it out, see receive.gui_script